index.js

```diff
@@ -1497,6 +1498,23 @@
     return xBool !== yBool ? or(x, y) : xEmpty;
   });
 
+  //# ifElse :: (a -> Boolean) -> (a -> b) -> (a -> b) -> a -> b
+  //.
+  //. Takes a unary predicate, two unary functions and a value of any type
+  //. and applies the first function to the value if the predicate evaluates
+  //. to true; otherwise applies the second function to the value.
```

Comment:
We could make it clear that the return value is the result of applying one of the `a -> b` functions. Perhaps:

_Takes a unary predicate, a unary "if" function, a unary "else" function, and a value of any type, and returns the result of applying the "if" function to the value if the value satisfies the predicate; the result of applying the "else" function to the value otherwise._